src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/AldersvilkårVurderingTest.kt

```diff
@@ -55,7 +47,16 @@ class AldersvilkårVurderingTest {
     }
 
     @Test
-    fun `Gyldige perioder med målgruppe DAGPENGER skal gi svar NEI`() {
+    fun `Gyldige perioder med målgruppe OVERGANGSSTØNAD skal kaste feil`() {
+        val målgruppe = dummyVilkårperiodeMålgruppe().copy(type = MålgruppeType.OVERGANGSSTØNAD)
+        val grunnlagsdata = grunnlagsdataDomain()
+
+        val feil = assertThrows<Feil> { vurderAldersvilkår(målgruppe, grunnlagsdata) }
+        assertThat(feil.message).isEqualTo("Aldersvilkår vurderes ikke for målgruppe: OVERGANGSSTØNAD")
+    }
+
+    @Test
+    fun `Gyldige perioder med målgruppe DAGPENGER skal kaste feil`() {
```

Comment:
Er vi sikre på at det ikke er en aldersvurdering som bør gjøres på dagpenger? Det er ikke krise akk nå fordi det er ikke en målgruppe som kan brukes før vi kommer til reisestønadene, men bør kanskje noteres ned et sted at det bør fikses om de har aldersvilkår

Reply:
DAGPENGER er iallefall ikke med som stønadstype i Favro-oppgaven, og har heller ikke vært et tema for denne oppgaven: https://favro.com/organization/98c34fb974ce445eac854de0/4d617346d79341c7fbd9a40a?card=NAV-23528

Hvor er et egnet sted for å notere dette?